environments/argocd-app-templates/appsetup-int.yaml

```diff
@@ -0,0 +1,38 @@
+###############################################################
+# Copyright (c) 2024 BMW Group AG
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dim-wrapper
+spec:
+  destination:
+    namespace: product-portal
+    server: 'https://kubernetes.default.svc'
+  source:
+    path: charts/dim
+    repoURL: 'https://github.com/phil91/dim-client.git'
+    targetRevision: chore/349-deploy-to-eV
```

Comment:
is this planned to stay like this? or are we changing it to an tag?

Reply:
yes, will be changed with release of the next version